src/juris-at-desc.json

```diff
@@ -32,13 +32,13 @@
       "name": "Handelsgericht",
       "abbrev": "HG >"
     },
-    "vgwg": {
+    "vgw": {
       "name": "Verwaltungsgericht",
```

Comment:
The abbreviation is in fact for **V**erwaltungs**g**ericht **W**ien, thus, should the name be changed to "Veraltungsgericht Wien" accordingly?